dmytroxIntel/sonic-buildimage
# Align with onie eeprom

#### Why I did it
BMC doesn't provide all platform information, so `show version` was not showing all parameters.
#### How I did it
Expose Onie system eeprom for sonic parser framework via thrift platform manager service.
#### How to verify it
Run `show version` command, all parameters are visible.
Verify Eeprom contents in redis STATE_DB parsed by syseepromd.
#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
